hw1/02-palindrome.html

```diff
@@ -8,20 +9,26 @@
       rel="stylesheet"
       href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
     <script defer src="02-palindrome.js"></script>
+    <link rel="stylesheet" href="styles.css">
   </head>
   <body>
     <main>
-      <section>
+    <div class="col-md-6 mx-auto mt-4"> 
+      <section class = "box-section">
         <h1>Exercise 02 - Palindrome</h1>
         <div>
           <label>Enter a positive number:</label>
```

Comment:
This label was flagged by WAVE extension by not being associated with a form input.